site/content/docs/developer/creating-new-addon-in-zap-extensions.md

```diff
@@ -0,0 +1,154 @@
+---
+title: "Creating a New Add-on in zap-extensions"
+description: "This guide will get you started with building and running ZAP in the Eclipse IDE."
+tags: 
+- guide
+- tutorial
+type: page
+date: "2021-03-19"
+---
+
+ZAP has a plugin architecture and new functionality is implemented via add-ons.
+
+Add-ons can be defined in any repository but most of the ones that the ZAP core team maintains live in 
+[zap-extensions](https://github.com/zaproxy/zap-extensions/).
```

Comment:
Should this be more clear that's for add-ons that are intended to be permanently added to the `zap-extensions` repo?